src/lib/libdylink.js

```diff
@@ -715,7 +715,14 @@ var LibraryDylink = {
           }
           if (prop in wasmImports && !wasmImports[prop].stub) {
             // No stub needed, symbol already exists in symbol table
-            return wasmImports[prop];
+            var res = wasmImports[prop];
+#if JSPI
```

Comment:
Do any tests fail if you make this `if ASYNCIFY` (if only to symmetry with below)

Reply:
I doubt it, let's see.